tutorials/plugins/editor/making_plugins.rst

```diff
@@ -177,7 +177,7 @@ clicked. For that, we'll need a script that extends from
     {
         public override void _EnterTree()
         {
-            Connect("pressed", this, "clicked");
+            Connect("pressed", clicked);
```

Comment:
Note that I didn't test the C# version.
The GDScript works

Reply:
CC @godotengine/mono

Reply:
In C# the naming convention for methods is PascalCase so the method `clicked` should be named `Clicked`.

And in Godot 4.0 signals are generated as C# events by the bindings generator so you can connect them like you would any other C# event. The syntax is:

```csharp
Pressed += Clicked;
```

I haven't checked what name is generated for the event because in this case I think it conflicts with the `Pressed` property so it probably would be generated with a suffix like `PressedSignal` or something.

Reply:
Cool, I'll rename the method to follow the C# naming convention. Tks

Reply:
You still need to change the connect syntax in C#:

```csharp
Pressed += Clicked;
```